appveyor.yml

```diff
@@ -21,8 +21,6 @@ install:
   - go env
   - set CERT= & set CERTPASS= & set
 build_script:
-  - go get github.com/motemen/go-cli/gen
-  - go generate
   - go get golang.org/x/tools/cmd/stringer
```

Comment:
`stringer` is also needless anymore (since #281)

Reply:
I know. I did it at #466.